src/ng/sniffer.js

```diff
@@ -61,7 +65,7 @@ function $SnifferProvider() {
       // so let's not use the history API also
       // We are purposefully using `!(android < 4)` to cover the case when `android` is undefined
       // jshint -W018
-      history: !!($window.history && $window.history.pushState && !(android < 4) && !boxee),
+      history: !!(hasHistoryPushState && !(android < 4) && !boxee),
```

Comment:
why move the `$window.history....` checks out of this expression? Is Chrome more special that `android < 4` or `boxee`?

I would expect it is clearer to simply change this to:

```js
!!(!isChromePackagedApp && $window.history && $window.history.pushState && !(android < 4) && !boxee),
```